Src/DependencyCollector/Microsoft.AI.DependencyCollector/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "2.3.0-beta2",
```

Comment:
Is there any way to pick this up from the targets or props file where it is defined now so we don't have to remember to maintain it in two places?
If not, please update to 2.3.0-beta3 since that just shipped and update the OneNote instructions for incrementing the version number for a release to include this.